### DISCUSSION TITLE
项目发布下载

### DISCUSSION THREAD

**Discussion by chinaxujia** [illacloud/illa-builder#1716] [Q&A]
我想咨询一下，项目构建完成后，能否直接下载html、js、css文件包？
我有自己的后台系统，并提供了rest接口，可以与illa的组件打通，所以我想通过illa构建完成后，把网页直接复制到自己的后台系统运行，不依赖于illa自身的web服务。

**Comment by donqu1xotechina** [comment]
Hey 我们在线上会议聊过了，我们会考虑的，感谢分享您的需求。
